Source/ActionController.swift

```diff
@@ -253,6 +258,13 @@ open class ActionController<ActionViewType: UICollectionViewCell, ActionDataType
             }
         }
         
+        // register footer
+        if settings.cancelView.hasCancelView, settings.cancelView.displayInFooter, cancelView != nil {
+            let nib = UINib(nibName: "CancelCell", bundle: Bundle(for: CancelCell.self))
+            collectionView.register(nib, forSupplementaryViewOfKind: UICollectionElementKindSectionFooter , withReuseIdentifier: ReusableViewIds.SectionFooter.rawValue)
+            // Subscribe to cancel cell tap
+            collectionView.delegate = self
```

Comment:
I think that this can be set when creating the collectionView. Seems reasonable to me that the collection view's delegate be always the `ActionController`. What do you think?